### PR TITLE
[PROPOSAL] Hook TeraVM to CI

### PR DESCRIPTION
**Pull request by tmdzk**
Signed-off-by: Timothée Dzik <timdzik@fb.com>

[PROPOSAL] Hook TeraVM to CI

## Summary

[PROPOSAL] Hook TeraVM to CI

## Test Plan

none

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/proposals/p006_hook_terravm_to_ci.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+|  +-----------v+  +--------v-----+   +------v------+ |
+|  |            |  |              |   |             | |
+|  |            |  |              |   |             | |
+|  |  AGW       |  |   FEG        |   |   NG40      | |
+|  |            |  |              |   |             | |
+|  |            |  |              |   |             | |
+|  +------------+  +--------------+   +-------------+ |
+|                                                     |
++-----------------------------------------------------+
+
```
Comment (uri200) [inline docs/readmes/proposals/p006_hook_terravm_to_ci.md:192]: in design 2, how will we receive the reports back from teravm?

File: docs/readmes/proposals/p006_hook_terravm_to_ci.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+- New components are thrown in the picture (Tiplab Bastion)
+- More dev time
+- More maintenance
+
+
+#### Design 3
+
+Using TeraVM cloud (AWS). No design for this option as I don't know the
+feasibility of it.
```
Comment (uri200) [inline docs/readmes/proposals/p006_hook_terravm_to_ci.md:247]: in this option we should factor in the cost of AWS. Teravm would use lots of cpu cycles so not sure if it is worth it the cost when we already have the infra in place

**Review by uri200 — APPROVED** [review]

File: docs/readmes/proposals/p006_hook_terravm_to_ci.md (hunk 2 above)
Comment (tmdzk) [reply docs/readmes/proposals/p006_hook_terravm_to_ci.md:247]: We also need information about who is hosting it etc .. that's why I left it "blank"

File: docs/readmes/proposals/p006_hook_terravm_to_ci.md (hunk 1 above)
Comment (tmdzk) [reply docs/readmes/proposals/p006_hook_terravm_to_ci.md:192]: phase 1-4 is to just trigger it, nice reporting will occur in phase 5

**Comment by tmdzk** [comment]
Moving to a different document.

**Pull request closed without merging**